fvm/transactionStorageLimiter.go

```diff
@@ -60,3 +92,13 @@ func (_ TransactionStorageLimiter) CheckStorageLimits(
 
 	return nil
 }
+
+func (d TransactionStorageLimiter) getCommonAddressAndStorageUsed(env environment.Environment, address flow.Address) (common.Address, uint64, error) {
+	c := common.Address(address)
+	u, err := env.GetStorageUsed(c)
+	if err != nil {
+		err = fmt.Errorf("storage limit check failed: %w", err)
```

Comment:
wrap the error around CheckStorageLimits instead of getCommonAddressAndStorageUsed to consistently report "storage limit check failed"

Also, maybe drop this method since it nothing after you've move the wrap